### DISCUSSION TITLE
Switch build?

### DISCUSSION THREAD

**Discussion by felipezacani** [yancharkin/SpelunkyClassicHD#13] [Q&A] [answered]
Any possibility of generating an updated switch build?

Thanks!

**Comment by yancharkin** [comment]
No.

**Comment by yancharkin** [comment, marked as answer]
I'm stopping the development, so I decided to give a proper answer to this question.

**Short answer.**
In theory, it's possible, but obviously, I'm not going to do it.

**Longer answer.**

You need to:
- own Nintendo Switch
- purchase GMS 2 license
- purchase Nintendo Switch license or GMS 2 Ultimate license
- become a Nintendo Switch developer and gain access to Nintendo Switch SDK (likely you have to pay for it)
- port the game to GMS 2 (and possibly make adjustments specifically for the Switch)

You have to spend a considerable amount of time and money. Maybe it is possible to use cracked software, maybe not (I'm against piracy). But you have to do a lot of work anyway. [There is a person who tried to port the game.](https://github.com/osum4est/spelunky-switch) I don't know if he succeeded.
